docs/mesos/DCOS.md

```diff
@@ -0,0 +1,134 @@
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Installing Calico in DCOS
+This guide provides instructions for installing Calico and its
+prerequisites on a Mesos DCOS cluster.
+
+##Prerequisites
+
+### Mesos Master and Agents
+
+You will need to be running a cluster with at least one Mesos master
+and and at least one Mesos agent (though we recommend having two or
+more agents).
+
+We currently support Mesos version `v0.28.0` and on for use with DCOS.
+Additionally, we support Centos7 as an operating system for use with
+the Unified Containerizer and Docker Containerizer, though additional
+OS's *may* work (though have not been fully tested!) with the Docker
+Containerizer.
+
+### Etcd
+Calico uses etcd as its datastore.
+
+To install etcd:
+ - Go to the **Universe** tab in your DCOS UI.
+ - Find the **etcd** package
+ - Click the **Install** button
+ - (Optional) choose **Advanced Installation** from
+the pop-up window.
+	- In non-production environments used for testing, set all of
+      the **mem** options to use 50MB or lower (this includes
+      `mem` and `mem-limit`).
+ - Choose **Review and Install**
+ - Double-check configuration then click **Install**.
+
+You can check the status of etcd by visiting the **Services**
+tab. You can install Calico once etcd appears with a status
+of **`Healthy`** (note that this could take a few minutes).
+
+## Install Calico
+You can easily install Calico from within the DCOS UI.
+
+### Get Calico Package
+You will need to link the Calico package from our git
+repository before you can install it.  To do this:
+
+ - Click the **System** tab
+ - Select **Add Repository** in the upper right
+ - Enter a name such as "Calico" for the package
+ - Paste the following git zip archive as the package link:
+   `https://github.com/robbrockbank/universe/archive/calico-universe.zip`
+
+### Install Calico Package
+You can now install the Calico package from the **Universe**
+tab. To start the install:
+ - Go to the **Universe** tab
+ - Find the Calico package
+ - Click the **Install** button then choose **Advanced
+   Installation** in the pop-up window.
+
+![alt tag](../images/mesos/dcos-calico-package-install.png)
+
+##### At this point you have a few options:
+
+In non-production environments used for testing, 
+you can set each memory limit option to have a
+value of 50MB as seen in the image below. This will
+ensure that smaller tasks with low memory requirements
+will be scheduled immediately. The changes include
+modifying `mem-limit-framework`, `mem-limit-install`,
+`mem-limit-etcd-proxy`, `mem-limit-node`, and
+`mem-limit-libnetwork`.
+
+![alt tag](../images/mesos/dcos-calico-config-changes.png)
+
+You could also speed up the install by increasing
+`max-concurrent-restarts`, allowing Calico to restart
+the Mesos agent and Docker services on more than one
+agent at a time.
```

Comment:
You should probably addd a big fat warning about this though.  Not generally recommended - but fine if you're turning up a new system.